priv/static/specs/http_api_spec.yaml

```diff
@@ -79,3 +134,59 @@ definitions:
     properties:
       version:
         type: "string"
+  SessionsList:
+    type: "object"
+    properties:
+      TotalCount:
+        type: integer
+      Sessions:
+        type: array
+        items:
+          $ref: '#/definitions/SessionInfo'
+  SessionInfo:
+    type: "object"
+    properties:
+      IMSI:
+        type: string
+      IMEI:
+        type: string
+      MSIDN:
+        type: string
+      Version:
+        type: string
+      RemoteControlIp:
```

Comment:
I prefer IP in all captial letters, @mgumz what is the policy in out other API's?

Reply:
@RoadRunnr i prefer IP as well. it should be treated similar to IMEI, IMSI, GTP etc.